Sources/SwiftKuery/Table.swift

```diff
@@ -36,6 +59,7 @@ open class Table: Buildable {
         return alias ?? _name
     }
     
+    // MARK: Initializer
     /// Initialize an instance of Table.
```

Comment:
backtick `Table`

Reply:
On line 82 please change /// Initialize an instance of Table with a table name and an array of Columns
to backtick Table and add a full stop.